clients/src/main/resources/common/message/MetadataRequest.json

```diff
@@ -16,9 +16,9 @@
 {
   "apiKey": 3,
   "type": "request",
-  "listeners": ["zkBroker", "broker", "controller"],
+  "listeners": ["zkBroker", "broker"],
   "name": "MetadataRequest",
-  "validVersions": "0-11",
+  "validVersions": "0-12",
```

Comment:
I am not sure if just modifying the valid versions bumps the version number (or if we want to bump the version at all)

Reply:
We shouldn't need to bump the version here if all we're changing is `listeners`.

Reply:
Makes sense. Will remove the version bump.